cudapoa/samples/sample_cudapoa.cpp

```diff
@@ -203,7 +201,7 @@ int main(int argc, char** argv)
     if (long_read)
     {
         const std::string input_file = std::string(CUDAPOA_BENCHMARK_DATA_DIR) + "/sample-bonito.txt";
-        generate_window_data(input_file, 8, 6, windows, batch_size);
+        generate_window_data(input_file, 55, 6, windows, batch_size);
```

Comment:
Do we need 55 specifically, or is that all the windows in the file? if you send `number_of_windows` as `-1` to the `parse_windows_file` function it returns all the windows in that file.

Reply:
That's total number of windows. I wrote it consistent with the short-reads that explicitly says 1000, will change both to -1 then.

Reply:
on a second thought, since it's sample, to demonstrate an example of both use cases, I changed back the short reads to 1000.